src/List.tsx

```diff
@@ -1,40 +1,51 @@
-import Item from "./Item";
+import { ListItem } from "./App";
+import {makeStyles} from 'tss-react'
+
+interface ListProps {
+  title: string;
+  items: Array<ListItem>;
+  handleCheck: (item: ListItem, checked: boolean) => void;
+}
+
+const useStyles = makeStyles()(() => ({
+  container: {
+    display: "flex",
+    flexDirection: "column",
+  }
+}))
+
+function List({ title, items, handleCheck }: ListProps) {
+  const { classes } = useStyles();
 
-function List({ title, items, handleCheck }) {
   return (
-    <div
-      style={{
-        display: "flex",
-        flexDirection: "column",
-      }}
-    >
+    <div className={classes.container}>
+      <h2 style={{ color: "red", textAlign: "center" }}>
+        {title}
+      </h2>
+      <p style={{ textAlign: "center" }}>
+        Number of items: {items.length}{" "}
+      </p>
       <div
         style={{
           display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
           flexDirection: "column",
           border: "solid 2px black",
           padding: "50px",
           borderRadius: "10%",
+          height: 250,
+          width: 250,
+          overflow: 'scroll'
```

Comment:
Note that I gave fixed height and width to this container, so that they won't grow or shrink according to their content. This is important because this behavior badly influences UI/UX.

Reply:
Make sense, i just want to style scrollbar a bit more